modules/standard/IO.chpl

```diff
@@ -2772,11 +2781,10 @@ This function is equivalent to calling :proc:`open` and then
               corresponding parameter of the :record:`channel` type.
               Defaults to true, but when safe, setting it to false
               can improve performance.
-:arg start: zero-based byte offset indicating where in the file the
-            channel should start reading. Defaults to 0.
-:arg end: zero-based byte offset indicating where in the file the
-          channel should no longer be allowed to read. Defaults
-          to a ``max(int(64))`` - meaning no end point.
+:arg region: zero-based byte offset indicating where in the file the
+            channel should start and stop reading. Defaults to
+            ``0..max(int(64))``, meaning from the start of the file to no
```

Comment:
Now that we are using ranges here, I think we could just say ``0..`` without specifying ``max(int(64))``.